variable_set_test.go

```diff
@@ -232,6 +235,7 @@ func TestVariableSetsRead(t *testing.T) {
 }
 
 func TestVariableSetsUpdate(t *testing.T) {
+	skipUnlessBeta(t)
```

Comment:
I have a slight worry that when we add a skip to an existing test it can become skipped for too long and my preference is that we create a second test for "enforced" or leave out the existing test until the beta label is removed. On the other hand, you've been really good about un-beta-ing features in the past. I'll leave it up to you based on what you know about this new feature.

Reply:
This feature will be out within 3 to 4 weeks, but since it is a major feature I agree with you.